src/SFA.DAS.Apprenticeships.Web/SFA.DAS.Apprenticeships.Web.csproj

```diff
@@ -22,10 +22,12 @@
     <PackageReference Include="SFA.DAS.Provider.Shared.UI" Version="2.0.30" />
     <PackageReference Include="System.Configuration.ConfigurationManager" Version="8.0.0" />
     <PackageReference Include="System.Drawing.Common" Version="8.0.0" />
+    <PackageReference Include="System.Formats.Asn1" Version="8.0.1" />
```

Comment:
what is this?

Reply:
Package vulnerability, Its actually part of the other ticket, I just built off the net8 branch